brave-unbreak.txt

```diff
@@ -88,3 +87,36 @@
 ||pdfjs.robwu.nl
 ! Allow 1P piwik since it's no different than server logs
 @@https://analytics.*/piwik.$~third-party
+! Fully block Facebook everywhere just like our Disconnect blocking but unbreak logins like from quora.com and twitch.tv
+! Note that options will be added to exclude these filters soon.
+||facebook.com$third-party
+||facebook.net$third-party
+||staticxx.facebook.com$third-party
+@@||connect.facebook.com/*/sdk.js$script
+@@||connect.facebook.net/*/sdk.js$script
+@@||facebook.com/connect/
+@@||www.facebook.com/connect
+@@||staticxx.facebook.com/connect/
+@@||graph.facebook.com/
+! Block fbcdn.net everywhere but allow Facebook embeds
+||fbcdn.net$third-party,domain=~facebook.com
+@@||staticxx.facebook.com/
+@@||xx.fbcdn.net/
+@@||www.facebook.com/*/plugin
+@@||www.facebook.com/plugins/
```

Comment:
If we unblock all of the plugins, then we'll let through the Like/Share buttons that @tomlowenthal wanted us to block in brave/brave-browser#3241:

- `www.facebook.com/plugins/like.php`
- `www.facebook.com/plugins/share_button.php`